R/Study_Report.R

```diff
@@ -58,6 +58,7 @@ Study_Report <- function(
   # input check
   lAssessments <- if("lStudyAssessResults" %in% names(lSnapshot)){lSnapshot$lStudyAssessResults} else {lSnapshot}
   lStatus <- if("lStatus" %in% names(lSnapshot)){lSnapshot$lStatus} else {NULL}
+  lLongitudinal <- if(exists("lStackedSnapshots", where = lSnapshot)){lSnapshot$lStackedSnapshots} else {NULL}
```

Comment:
This can wait for another PR or you can add now, but I think we should add a section here where if the user chooses `strReportType` == "QTL", we should check if a longitudinal model was passed through and throw an error if not. 

I tried running this with a single snapshot (incorrectly), and ran into an error while trying to render the report, but think we can throw an error about this. 

Something like this (not tested):
```
if (strReportType == "QTL") {
  if (exists("lStackedSnapshots", where = lSnapshot)) {
    lLongitudinal <- lSnapshot$lStackedSnapshots
  } else {
    cli::cli_abort("QTL reports require a longitudinal data model.")
  }
}
```